apps/console/src/features/users/components/user-change-password.tsx

```diff
@@ -199,7 +215,7 @@ export const ChangePasswordComponent: FunctionComponent<ChangePasswordPropsInter
                     }
                 }
             ],
-            "schemas": ["urn:ietf:params:scim:api:messages:2.0:PatchOp"]
+            "schemas": [ "urn:ietf:params:scim:api:messages:2.0:PatchOp" ]
```

Comment:
remove the additional spaces

Reply:
These are required by the ES-lint code style formatter